pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/ReactiveMessageConsumer.java

```diff
@@ -25,9 +25,9 @@
 
 public interface ReactiveMessageConsumer<T> {
 
-	<R> Mono<R> consumeMessage(Function<Mono<Message<T>>, Mono<MessageResult<R>>> messageHandler);
+	<R> Mono<R> consumeOne(Function<Mono<Message<T>>, Publisher<MessageResult<R>>> messageHandler);
```

Comment:
Time to add javadoc ?
(repeat for all API methods)

Reply:
Yes, the javadoc needs to be addressed too.

Reply:
I'd rather do that separately so that we get the API structure in a state where we can agree on it.